CONTRIBUTING.md

```diff
@@ -1,139 +1,79 @@
 
-    We strive to have high and useful coverage by unit tests.  If your change
-    involves substantial logic, we will request that you write applicable unit
-    tests.
+         # Clone your forked repository into local directory.
+         git clone FORKED_REPOSITORY_URL
 
-    Our unit tests are written with google-apputils basetest framework.
-    See a [basic example] of how to use it, in the "Google-Style Tests" section.
+         # Navigate to your newly cloned directory.
+         cd <path to you local cloned repository>
 
-    [Instructions to run the tests.]
+         # Assign the original repository to a remote called "upstream".
+         # Using HTTPS
+         git remote add upstream https://github.com/GoogleCloudPlatform/forseti-security.git
 
-7. Commit your changes and push them to your development branch.
+         # Using SSH
+         git remote add upstream git@github.com:GoogleCloudPlatform/forseti-security.git
 
-    ```
-    $ git push origin <my_development_branch>
-    ```
+         # Verify new upstream remote is added correctly.
+         git remote -v
 
-8. Open a Pull Request to begin the process for code review.
+         origin  git@github.com:YOUR_FORK (fetch)
+         origin  git@github.com:YOUR_FORK (push)
+         upstream  git@github.com:GoogleCloudPlatform/forseti-security.git (fetch)
+         upstream  git@github.com:GoogleCloudPlatform/forseti-security.git (push)
 
-    All submissions, including submissions by project members, require a code review.
-    To begin the review process, create a new GitHub pull request. The GitHub UI
-    will show if there are any merge conflict(s) to be resolved.
+  1. Fetch the latest changes from upstream into your cloned repository:
 
-    The GitHub Pull Request UI will show you dropdowns to select the destination of
-    your pull request:
-    * the base fork is the upstream
-    * the head fork is your fork with your changes
+         git fetch upstream
+         git checkout master
+         git merge upstream/master
 
-    All tests must pass before we will review your PR.
-    We also check the codebase with pylint. You can install the git hooks provided
-    under scripts/githooks/ to do the check locally.
+  1. Create a new local development branch for your feature or bug fix:
 
-9. Merging your PR.
+         git checkout -b YOUR_DEVELOPMENT_BRANCH
 
-    Once your PR is approved and all merge conflicts are resolved, we will merge your PR.
+  1. Create your change.
+    - Don't incorporate multiple changes in one PR. A change should be a logical, self-contained unit of work, feature, or fix. This simplifies troubleshooting and rollbacks.
+    - Learn how to execute [Inventory](inventory-quickstart), [Scanner](scanner-quickstart), or [Enforcer](enforcer-quickstart).
+  1. Create your test.
+    - Unit tests should provide high and useful coverage. If your change involves substantial logic, we'll ask you to write applicable unit tests.
```

Comment:
2nd sentence: "You should write applicable unit tests for your changes, especially for changes involving substantial logic."